inc/agl.inc

```diff
@@ -15,8 +15,19 @@ XT_QUIRK_UNPACK_SRC_URI += " \
 "
 
 # these layers will be added to bblayers.conf on do_configure
-XT_QUIRK_BB_ADD_LAYER += " \
-    meta-xt-images-domx \
```

Comment:
why do yo drop meta-xt-images-domx?

Reply:
please move meta-xt-images-domx to its original place

Reply:
Done